server.py

```diff
@@ -66,11 +66,17 @@ def process_data(data, connection: socket.socket, user, index):
     elif data[0] == "list":
         return database.list_stocks(user) if not user[3] else database.list_stocks_root()
     elif data[0] == "balance":
+        # Check command
+        if (len(data) != 1):
```

Comment:
Do we need these? Wouldn't it not enter the elif data[0] == 'balance'?

Reply:
its within the elif data[0] == 'balance' block? Just to show an error if anything is typed after 'balance'

Reply:
Ahhh I see. Cool